src/components/Organization/View.tsx

```diff
@@ -33,11 +33,15 @@ const OrganizationView = () => {
   const [loaded, setLoaded] = useState<boolean>(false)
   const [error, setError] = useState<string>()
   const [finished, setFinished] = useState<boolean>(false)
+  // we need refobserver to be in state to ensure the observer is assigned when rendering the ref layer
+  // otherwise, the observer is not assigned and the intersection is not triggered
```

Comment:
I should read better the entire sentence when written by copilot 👀 kinda cacophonous